frontend/src/metabase/querying/utils/drills/combine-columns-drill/components/ColumnAndSeparatorRow/ColumnAndSeparatorRow.module.css

```diff
@@ -0,0 +1,18 @@
+.separator {
+  flex: 0 0 auto;
+}
+
+.column {
+  flex: 1;
+}
+
+.remove {
+  flex: 0 0 auto;
+  border-color: transparent !important;
+}
+
+.wrapper {
+  &:not(:only-child) {
+    margin-top: 0;
```

Comment:
We have `marginTop` defined for most Mantine inputs by default.
I think it's because of the assumption that if the input is `:not(:only-child)` there will be a label above it.
This assumption does not hold for `Select` because it never is the only child due to `<input type="hidden"` being rendered before it (see [screenshot](https://github.com/metabase/metabase/assets/6830683/ba484c9a-99fe-400b-9823-d20984b195fd)).

It'd be better to assume that every label that is `:not(:only-child)` will have an input underneath, and add `marginBottom` to labels instead. Though it'd be ideal if the low-level building blocks never made such assumptions at all.

Anyway, fixing that is out of the scope of this PR, so I added this band-aid here.

Reply:
Fixed here: https://github.com/metabase/metabase/pull/41728